plugin-unmanaged-racks/config/config.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################################################
+# URP Server                                                                                       #
+#--------------------------------------------------------------------------------------------------#
+# Following section configures server where URP's REST API is exposed                              #
+####################################################################################################
+Host: odimra.local.com
```

Comment:
should Host be hardcoded? odimra takes the fqdn that we provide while deployment.

Reply:
As with other components this can/will be overwritten at install time. So I dont see any issue. Pls close this if you agree.

Reply:
`build` step copies this config into build directory. Copied config file contains host provided by HOSTIP environment variable.